common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -28,88 +25,85 @@ class ProjectCard extends React.PureComponent<Props> {
+            target="_blank" rel="noopener noreferrer">
+            {this._renderLogo()}
+            {this._renderSubInfo()}
+            {this._renderTitleAndIssue()}
+            {this._renderProjectDescription()}
+            {this._renderSkillsNeeded()}
+          </a>
         </div>
       </div>
     );
   }
-
-  _renderIssueAndLocation(): React$Node {
+  _renderLogo(): React$Node {
     return (
-      <div className="ProjectCard-issueAndLocation">
-        {
-          this._renderLabelAndValue(
-            'Issue Area: ',
-            this.props.project.issueArea,
-          )
-        }
-        { 
-          (this.props.project.location !== Locations.OTHER)
-          ? this._renderLabelAndValue('Location: ', this.props.project.location)
-          : this._renderLabelAndValue('Location: ', null)
-        }
-        {
-          this._renderDateModified()
-        }
+      <div className="ProjectCard-logo">
+        <img src={this.props.project && this.props.project.thumbnail ? this.props.project.thumbnail.publicUrl : '/static/images/projectlogo-default.png'}/>
       </div>
-    );
+    )
   }
-
-  _renderDateModified(): React$Node {
+  _renderTitleAndIssue(): React$Node {
     return (
-      <div className="ProjectCard-subtext">
-        <span className="ProjectCard-label">
-          Last Updated:&nbsp;
-        </span>
-        <span className="ProjectCard-value">
-          <Moment format="MMM D, YYYY, h:mm a">{this.props.project.date_modified}</Moment>
-        </span>
+      <div className="ProjectCard-title">
+        <h2>{this.props.project.name}</h2>
+        <h4>{this.props.project.issueArea}</h4>
       </div>
     )
   }
-
-  _renderLabelAndValue(label: string, value: string): React$Node {
-      return (
-        <div className="ProjectCard-subtext">
-          <span className="ProjectCard-label">
-            {label}
-          </span>
-          <span className="ProjectCard-value">
-            {value ? value : null}
-          </span>
+  _renderProjectDescription(): React$Node {
+    return (
+        <div className="ProjectCard-description">
+          <p>{Truncate.stringT(this.props.project.description, this.props.textlen)}</p>
         </div>
-      );
+    );
   }
-
-  _renderDescription(): string {
-    const maxDescriptionLength = 300;
-    return this.props.project.description.length > maxDescriptionLength
-      ? this.props.project.description.slice(0, maxDescriptionLength) + '...'
-      : this.props.project.description;
+  _renderSkillsNeeded(): React$Node {
+    return (
+    <div className="ProjectCard-skills">
+    <h3>Skills Needed</h3>
+      {this._generateSkillList(this.props.skillslen)}
+    </div>
+    )
+  }
+  _generateSkillList(numskills): React$Node {
+    //take array of skills needed from props, truncate if required, and map to list items
+    const skills = Truncate.arrayT(this.props.project.positions, numskills)
+    let index = 0; //super hacky unique key indexes goooo
```

Comment:
You can also get the index from the second parameter in the map function.
`skills.map((skills,i) => <li key={i}>{skills}</li>)`